recipes/genomic_address_service/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - pip
     - pytest-runner
   run:
```

Comment:
it looks like the dependencies are also out of date: https://github.com/phac-nml/genomic_address_service/blob/0d52d98146dba3b72bde4145f40639844d497dfc/setup.py#L51-L60

Reply:
something like:
```yaml
  - python >=3.10,<3.13
  - pyarrow >=14.0.0
  - numba ==0.59.1
  - numpy ==1.26.4
  - pytables ==3.9.1
  - six >=1.16.0
  - pandas ==2.0.2
  - pytest ==8.3.3
  - scipy ==1.14.1
  - psutil ==6.1.0
  - fastparquet ==2023.4.0
```

Reply:
Thanks so much. I appreciate the feedback. Fixed in [3feeda9](https://github.com/bioconda/bioconda-recipes/pull/54506/commits/3feeda9640ea92ffd6b681b701bac7d277896759)